src/AudioPlayer.cpp

```diff
@@ -850,11 +832,6 @@ void AudioPlayer_TrackQueueDispatcher(const char *_itemToPlay, const uint32_t _l
     gPlayProperties.currentTrackNumber = _trackLastPlayed;
     char **musicFiles;
 
-    #ifdef MQTT_ENABLE
-        publishMqtt((char *) FPSTR(topicLedBrightnessState), 0, false);
```

Comment:
@biologist79 I removed these 3 messages. It's not clear to me why the LED brightness or the playmode were sent here. If they were here for a good reason, we need to move them to the loop as well.